Libraries/CMSIS/Device/Maxim/MAX32690/Include/hpb_regs.h

```diff
@@ -176,8 +176,8 @@ typedef struct {
  * @ingroup  hpb_registers
  * @defgroup HPB_MEMBADDR HPB_MEMBADDR
  * @brief    Memory Base Address register. This is the base address of the addressable memory
- *           region in this microcontroller’s RAM. Because the addressable memory is mapped
- *           in 16M boundaries, the lower 24 bits are fixed to 0.
+ *           region in this microcontrollers RAM. Because the addressable memory is mapped in
```

Comment:
's was correct here.

Reply:
Had to revise this description. The ' was not supported in SVDConv.